### ISSUE TITLE
Documentation? 

### ISSUE THREAD

**Issue by kennes913** [pokitdok/gremlin-python#15]
I know this is a long shot since it looks like the last commit was almost 3 years ago, but what do you guys think about adding pythonic documentation?

**Comment by corbinbs** [comment]
Hi @kennes913 - are there some examples of what you'd like to see in the way of documentation?

The tests in https://github.com/pokitdok/gremlin-python/blob/master/tests/test_gremthon.py and https://github.com/pokitdok/gremlin-python/blob/master/tests/test_pipeline.py are likely the best "documentation" right now as far as Python examples/snippets for the different things that have been supported.  (beyond what's in https://github.com/pokitdok/gremlin-python#rexster )

I admit that I'm not using this library as much in day-to-day work.  I am definitely interested in what other folks may be doing with it so that we can make sure they have helpful documentation.

**Comment by cscetbon** [comment]
@corbinbs, which steps are available or which are not ? If there are that are not and the equivalent function name. For example, to use `as()` step we need to use `as_()`.

**Comment by corbinbs** [comment]
Hi  @cscetbon  - things should be named as expected in Gremlin except for those that conflict with Python keywords.   When there's a keyword conflict, you'll see that trailing `_` on the end of the name.

So things like:
* `in_`: https://github.com/pokitdok/gremlin-python/blob/master/gremthon.py#L331
* `property_`: https://github.com/pokitdok/gremlin-python/blob/master/gremthon.py#L356
* `except_`: https://github.com/pokitdok/gremlin-python/blob/master/gremthon.py#L392
* `or_`: https://github.com/pokitdok/gremlin-python/blob/master/gremthon.py#L403
* `as_`: https://github.com/pokitdok/gremlin-python/blob/master/gremthon.py#L575

I think those were the main things that had to be slightly adjusted to avoid naming hard times.

**Comment by cscetbon** [comment]
Yeah @corbinbs I understand. Just saying that'd be great if it's added to the README file.
